README.md

```diff
@@ -8,6 +8,39 @@ It can also be used as a standalone binary and library.
 
 You can find more information about the linter and it's implemented rules in our [documentation](https://mozilla.github.io/addons-linter/).
 
```

Comment:
The CircleCI is still pointing to the firefox repo, correct? If it does not work for us, I guess we can just remove it for now.

Reply:
There are more places in this readme, which need an update to reflect this is a different repo and not the firefox one. Since these places are not changes, the review tool does not allow me to add remarks there, or at least I am not able to add them. :-( 

For example the `Quick Start` section has the firefox repo link hardcoded.

Reply:
I'll adjust the readme. This will also be used on ATN.